src/components/split-button/split-button-test.stories.tsx

```diff
@@ -53,44 +54,39 @@ export default {
   },
 };
 
-/* eslint-disable react/prop-types */
+type SplitButtonStoryArgs = {
+  buttonType?: SplitButtonProps["buttonType"];
+  textContent: string;
+  text: string;
+  subtext: string;
+};
```

Comment:
Do we definitely only need those specific 3/4 props? I notice the story previously used `iconType`, `iconPosition`, `dataElement` and `dataRole` which we're now excluding. (While they will in theory still be passed down as part of `args`, using this type will mean that the Typescript compiler won't allow them to be supplied.)